gslib/commands/signurl.py

```diff
@@ -151,7 +157,9 @@
   -u           Use service account credentials instead of a private key file
                to sign the url.
 
-               You can equivalently use --use-service-account instead of -u
+               You can equivalently use --use-service-account instead of -u.
+               Note that the max alloweod duration is 12 hours if this option
+               is used.
```

Comment:
Might want to have @starsandskies weigh in on this one as there may be a better way to say this.

Reply:
"You can use --use-service-account instead of -u to [do this]." where [do this] is the benefit of using --use-service-account. If there's no difference at all between the two commands, maybe it would be more efficient to say "-u, --use-service-account" in the first column.

Is the max duration 12h for both -u and --use-service-account? If so, I suggest "The maximum allowed duration for the valid link is 12 hours if you use the --use-service-account or -u option."

Reply:
Regarding "You can equivalently use --use-service-account instead of -u", yes, there is no difference between the two options. We tried to fit it in the first column last time (cl/303730086) but it messes up with the formatting because of the length of --use-service-account.

Reply:
I see, thanks for the context. How about "You can also use the --use-service-account option, which is equivalent to -u. Note that both options have a maximum allowed duration of 12 hours for a valid link."

And it would be helpful to code format the options.